Integrations/integration-PhishMe.yml

```diff
@@ -457,4 +471,11 @@ script:
     - contextPath: DBotScore.Score
       description: The actual score
     description: Check if sender address is malicious
-hidden: false
+  - name: phishme-get-report
+    arguments:
+    - name: threat-id
+      required: true
+      default: true
+      description: Thread ID to retrieve report of.
+    description: 'Retrieves individual PhishMe Intelligence reports.'
+releaseNotes: 'Added new command to retrieve reports from PhishMe'
```

Comment:
include the name of the command in the releaseNotes `Added new command "phishme-get-report" to ....`